lib/crawly/request.ex

```diff
@@ -48,6 +48,7 @@ defmodule Crawly.Request do
     # incoming requests
     default_middlewares = [
       Crawly.Middlewares.DomainFilter,
```

Comment:
Not necessary, since it defaults to an empty list in the request struct anyway

Reply:
yeah, agreed